setup.py

```diff
@@ -1,63 +1,75 @@
 #!/usr/bin/env python
 # coding: utf-8
 
+# pylint: disable = C0103
+
+"""
+Packaging
+"""
+
 # Copyright (c) - Damian Avila
 
 from __future__ import print_function
 
-name = "rise"
+import sys
+import os
+from setuptools import setup, find_packages
 
-# Minimal Python version sanity check
+from rise._version import __version__ as version
 
-import sys
+NAME = "rise"
+
+# Minimal Python version sanity check
 
-v = sys.version_info
-if v[:2] < (2, 7) or (v[0] >= 3 and v[:2] < (3, 4)):
-    error = "ERROR: %s requires Python version 2.7 or 3.4 or above." % name
+PYVER = sys.version_info
+if PYVER[:2] < (2, 7) or (PYVER[0] >= 3 and PYVER[:2] < (3, 4)):
+    error = "ERROR: %s requires Python version 2.7 or 3.4 or above." % NAME
     print(error, file=sys.stderr)
     sys.exit(1)
 
 # Main
 
-import os
-from distutils.core import setup
-
 pjoin = os.path.join
 here = os.path.abspath(os.path.dirname(__file__))
-pkg_root = pjoin(here, name)
+pkg_root = pjoin(here, NAME)
 data_files = []
 
-packages = []
-for d, _, _ in os.walk(pkg_root):
-    if os.path.exists(pjoin(d, "__init__.py")):
-        packages.append(d[len(here)+1:].replace(os.path.sep, "."))
-
 paths = []
 for (path, directories, filenames) in os.walk(pjoin(pkg_root, "static")):
     target_dir = os.path.normpath(
-                          os.path.join('share/jupyter/nbextensions/rise',
-                          os.path.relpath(pjoin(path), pjoin(pkg_root, 'static'))))
+        os.path.join('share/jupyter/nbextensions/rise',
+                     os.path.relpath(pjoin(path), pjoin(pkg_root, 'static'))))
     files = []
     for filename in filenames:
         paths.append(os.path.relpath(pjoin(path, filename), pkg_root))
         files.append(os.path.relpath(pjoin(path, filename), here))
+    # nbconfigurator is configured to use this
+    if target_dir == 'share/jupyter/nbextensions/rise':
+        files.append('README.md')
     data_files.append((target_dir, files))
+
 data_files.append(('etc/jupyter/nbconfig/notebook.d', ['rise.json']))
 
-version_ns = {}
-with open(pjoin(here, name, '_version.py')) as f:
-    exec(f.read(), {}, version_ns)
+data_files
+
+INSTALL_REQUIRES = [
+    'notebook>=5.5',
+]
 
 with open('./README.md') as readme:
     README = readme.read()
 
+for d in data_files:
```

Comment:
debugging stuff? If you really want to show the content of the data_files object, I would put this piece after line 51, I think...